src/devcards.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DevCards from './components/DevCards.jsx';
```

Comment:
Just tells right away if you deal with a file containing JSX syntax or not. But I can very well change the extensions to `.js` if you want.